JarrydWorland/2021_Capstone_GP2_Team
# New merlin design

## Description
This PR updates Merlin to use the new character design.

## Changes Include
- New sprites
- New animations
- Shooting animation speeds up with rapid fire upgrades
- Player size and collider adjustments

## How To Test
Play through the game, since Merlin has changed ensure he is not getting stuck on any geometry, ensure the animations look correct in all scenarios.
